web/template/layout/edit-panel.gotmpl

```diff
@@ -16,48 +16,34 @@
           <h3 class="govuk-heading-s govuk-!-margin-0">Edit tasks</h3>
         </div>
       </header>
-          <div class="govuk-width-container">
-          <form action="{{ prefix "/" }}" method="POST" id="reassign-task-form" class="ignore-pagination-bottom">
-            <h4 class="govuk-heading-s"><span class="count-checked-checkboxes js-mt-task-count">0</span> tasks selected</h4>
-            <div class="govuk-form-group govuk-!-display-inline-block govuk-!-margin-right-5">
-              <label class="govuk-label" for="sort">
-                Assign to a team
-              </label>
-              <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"  class="js-xsrfToken"/>
-              <label for="assignTeam"></label> <select class="govuk-select option-value-select js-assign-team-select" data-module="assign-team-select" id="assignTeam" name="assignTeam">
-                <option value="0" selected>Select a team</option>
-                  {{ range .TeamSelection }}
-                  <option class="option-value" value="{{ .Id }}">{{ .Name }}</option>
-                  {{ end }}
-              </select>
-            </div>
+      <div class="govuk-width-container">
+        <div id="reassign-task-form" class="ignore-pagination-bottom">
+        <h4 class="govuk-heading-s"><span class="count-checked-checkboxes js-mt-task-count">0</span> tasks selected</h4>
+        <div class="govuk-form-group govuk-!-display-inline-block govuk-!-margin-right-5">
+          <input type="hidden" name="xsrfToken" value="{{ .XSRFToken }}"  class="js-xsrfToken"/>
+          <label class="govuk-label" for="assignTeam">Assign to a team</label>
+            <select class="govuk-select option-value-select js-assign-team-select" data-module="assign-team-select" id="assignTeam" name="assignTeam">
+            <option value="0" selected>Select a team</option>
+              {{ range .TeamSelection }}
+              <option class="option-value" value="{{ .Id }}">{{ .Name }}</option>
+              {{ end }}
+          </select>
+        </div>
 
-            <div class="govuk-form-group govuk-!-display-inline-block govuk-!-margin-right-5">
-              <label class="govuk-label" for="sort">
-                Assign to a case manager
-              </label>
-              <select class="govuk-select ignore-pagination-bottom" label for="selected-CM" id="assignCM" name="assignCM">
-                <option value="" label="Select a case manager" id="selected-CM" selected>Select a case manager</option>
-              </select>
-            </div>
+        <div class="govuk-form-group govuk-!-display-inline-block govuk-!-margin-right-5">
+          <label class="govuk-label" for="assignCM">
+            Assign to a case manager
+          </label>
+          <select class="govuk-select ignore-pagination-bottom" for="selected-CM" id="assignCM" name="assignCM">
```

Comment:
Worth removing the `for="selected-CM"` from the select, looks like it probably got there from a removed label in a diff...

Reply:
My bad, I think I misinterpreted what your comment meant

Reply:
Yeah, to be clear. You want to keep:

`<label class="govuk-label" for="assignCM">`

and remove the for from:

`<select class="govuk-select ignore-pagination-bottom" **for="selected-CM"** id="assignCM" name="assignCM">`

The select has no need of a for attr, but the label needs it to associate with the select.